roles/openshift_metrics/templates/hawkular_cassandra_rc.j2

```diff
@@ -0,0 +1,128 @@
+apiVersion: v1
+kind: Template
```

Comment:
Any reason this one remains a template?  Is it because its easier to process the PVC with the RC